src/rules.h

```diff
@@ -70,6 +71,8 @@ class HSClientChanges {
     bool            manage = true; // whether we should manage it
     bool            fullscreen;
     std::string     keymask; // Which keymask rule should be applied for this client
+
+    std::experimental::optional<int> pseudotile;
```

Comment:
Why is this an `int` and not `bool`?

Reply:
Because I'm not a clever man. Fix is coming up.